sdks/java/core/src/main/java/org/apache/beam/sdk/util/HistogramData.java

```diff
@@ -47,6 +47,9 @@ public class HistogramData implements Serializable {
   private long numTopRecords;
   private long numBottomRecords;
 
+  private double sum_of_squared_deviations;
```

Comment:
mark as with `@GuardedBy("this")` unless there are non threadsafe callers

Reply:
done